db/migrate/20170209192130_migrate_ansible_tower_configuration_manager_inventory_root_group_sti_type.rb

```diff
@@ -0,0 +1,21 @@
+class MigrateAnsibleTowerConfigurationManagerInventoryRootGroupStiType < ActiveRecord::Migration[5.0]
+  class EmsFolder < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time('Migrating STI type of ansible_tower inventory_root_groups to be of automation_manager') do
+      EmsFolder.where(:type => 'ManageIQ::Providers::AnsibleTower::ConfigurationManager::InventoryRootGroup').update_all(
```

Comment:
@bdunne this ^ `ManageIQ::Providers::AnsibleTower::ConfigurationManager::InventoryRootGroup` to `ManageIQ::Providers::AnsibleTower::AutomationManager::InventoryRootGroup`